impl/client/src/main/javascript/web/dojo/pentaho/common/PageControl.js

```diff
@@ -30,161 +30,185 @@
  * pc.registerPageNumberChangeCallback(controller.updatePageNumber);
  *
  */
- define(["dojo/_base/declare", "dijit/_WidgetBase","dijit/_TemplatedMixin", "dijit/_WidgetsInTemplateMixin","dijit/form/NumberTextBox", "dojo/dom-class", "dojo/keys"],
```

Comment:
Apparently, the file was completely reformatted to use 4 space tabs. While some files in pentaho/common use this rule, I'd argue they are the exception, and remains from past rules or lack thereof. Also, most other JS (and Java) codebases/areas use 2 space tabs, so I'd prefer to err on that side.

So, in this case, I would ask you to not do this. And to follow the surrounding formatting rules whenever possible.

Thanks!